frontend/src/metabase/containers/Form.jsx

```diff
@@ -227,8 +227,12 @@ class Form extends React.Component {
         const errorNames = Object.keys(error.data.errors);
         const hasUnknownFields = errorNames.some(name => !fieldNames.has(name));
         throw {
-          _error: hasUnknownFields ? t`An error occurred` : null,
-          ...error.data.errors,
+          _error:
+            error.data?.message ||
+            error.message ||
+            error.data ||
```

Comment:
Use the same condition as when there are no `errors` presented in the API response. Without these. The error will just be the generic one `An error occurred`